com.unity.netcode.gameobjects/Runtime/Serialization/IBufferSerializerImplementation.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    public interface IBufferSerializerImplementation
+    {
+        bool IsReader { get; }
+        bool IsWriter { get; }
+
+        ref FastBufferReader GetFastBufferReader();
+        ref FastBufferWriter GetFastBufferWriter();
+
+        void SerializeValue(ref object value, Type type, bool isNullable = false);
+        void SerializeValue(ref GameObject value);
```

Comment:
again, I'd advocate for killing these GO/NO/NB APIs in favor of #1173

Reply:
Done.